main/lsp/AutocorrectFlusher.cc

```diff
@@ -0,0 +1,19 @@
+#include "main/lsp/LSPTypechecker.h"
+
+using namespace std;
+namespace sorbet::realmain::lsp {
+
+void AutocorrectFlusher::flushErrors(spdlog::logger &logger, vector<unique_ptr<core::ErrorQueueMessage>> errors,
+                                     const core::GlobalState &gs, core::FileRef file) {
+    for (auto &error : errors) {
+        if (error->kind == core::ErrorQueueMessage::Kind::Error) {
+            if (error->error->isSilenced) {
+                continue;
+            }
+
+            prodHistogramAdd("error", error->error->what.code, 1);
```

Comment:
You can remove this call to `prodHistogramAdd`; I think the intent for this metric is to see which errors are getting reported to the user. These errors aren't being reported; we're just mining them for autocorrects.